NEWS.md

```diff
@@ -7,6 +7,12 @@
 - New functions `set_thumbnail()`, `get_thumbnail()`, `delete_thumbnail()` and
   `has_thumbnail()` let you interact with content thumbnails, replacing older
   `*_image()` functions. (#294, #304)
+- `groups_create_remote()` now has an `exact` parameter. Setting `exact` causes
```

Comment:
Should this mention anything about the change to `users_create_remote` as well?

Reply:
No — those changes were in a previous PR (from two years ago). The only change to `users_create_remote` was reverted.